src/main/resources/META-INF/plugin.xml

```diff
@@ -34,25 +34,34 @@
     <depends optional="true" config-file="pluginKotlin.xml">org.jetbrains.kotlin</depends>
 
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-    <idea-version since-build="201.7000"/>
+    <idea-version since-build="202.6000"/>
 
     <actions>
-        <action id="GoToRequestMapping" class="com.viartemev.requestmapper.GoToRequestMappingAction"
-                text="Request Mapping" description="Request mapping">
+        <action id="InboundRequestMapping" class="com.viartemev.requestmapper.actions.InboundRequestMappingAction"
+                text="Inbound Requests" description="Request mapping">
             <add-to-group group-id="GoToTargetEx" anchor="first"/>
             <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl BACK_SLASH"/>
         </action>
+        <action id="OutBoundRequestMapping" class="com.viartemev.requestmapper.actions.OutBoundRequestMappingAction"
+                text="Outbound Requests" description="Request mapping">
+            <add-to-group group-id="GoToTargetEx" anchor="first"/>
+            <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl o"/>
```

Comment:
Could we bind to another keystroke? This one was taken already by IDEA.

Reply:
Yes, of course. Could you help me? And could you say where you found this keystroke? I can't do it.